### PR TITLE
NR-289825:  Log Reporting: Console log level should be separate from Remote logging level,  NR-300193: Log Reporting: Use customCollector address

### PR DESCRIPTION
**Pull request by cdillard-NewRelic**
https://new-relic.atlassian.net/browse/NR-289825 

[iOS] [Phase 2] Log Reporting: Console log level should be separate from Remote logging level.

https://new-relic.atlassian.net/browse/NR-300193

[iOS] [phase 2] Log Reporting: Use customCollector address

### CODE REVIEW

**Comment by mbruin-NR** [comment]
Is there a way to add unit tests that prove that the log levels are respected differently?

**Comment by cdillard-NewRelic** [comment]
> Is there a way to add unit tests that prove that the log levels are respected differently?

I've added test named testRemoteLogLevels which prove the remote log level respected differently.

**Review by mbruin-NR — APPROVED** [review]

File: Agent/Measurements/NRMAMeasurementType.m [hunk 1]
```diff
@@ -5,35 +5,3 @@
 //  Created by Bryce Buchanan on 9/26/13.
 //  Copyright © 2023 New Relic. All rights reserved.
 //
-
```
Comment (smalsam-newr) [inline Agent/Measurements/NRMAMeasurementType.m:8]: What was the reason for removing this?

**Review by smalsam-newr — APPROVED** [review]

File: Agent/Measurements/NRMAMeasurementType.m (hunk 1 above)
Comment (cdillard-NewRelic) [reply Agent/Measurements/NRMAMeasurementType.m:8]: I noticed it wasn't being used but it showed up in a crash stack i was looking at. I figured its best to remove the dead dead code if its a safe removal like this.

**Pull request merged**